File: app/src/main/java/com/BlizzardArmory/starcraft/profile/Campaign.java

package com.BlizzardArmory.starcraft.profile;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * The type Campaign.
 */
public class Campaign {

    @SerializedName("difficultyCompleted")
    @Expose
    private DifficultyCompleted difficultyCompleted;

    /**
     * Gets difficulty completed.
     *
     * @return the difficulty completed
     */
    public DifficultyCompleted getDifficultyCompleted() {
        return difficultyCompleted;
    }

    /**
     * Sets difficulty completed.
     *
     * @param difficultyCompleted the difficulty completed
     */
    public void setDifficultyCompleted(DifficultyCompleted difficultyCompleted) {
        this.difficultyCompleted = difficultyCompleted;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/starcraft/profile/Campaign.kt
package com.BlizzardArmory.model.starcraft.profile

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


/**
 * The type Campaign.
 */
@Keep
data class Campaign(

    @SerializedName("difficultyCompleted")
    var difficultyCompleted: DifficultyCompleted

)